DataScience/AisAudioLabeler.py

```diff
@@ -509,25 +513,34 @@ def plot_intervals(shp, hmd):
     None
 
     """
-    fig, axs = plt.subplots()
+    fig, axs = plt.subplots(figsize=(10,9), dpi=100)
 
     # Consider each ship
     n_ship = 0
+    label_set = set()
```

Comment:
It seems you are using a label_set and the `if label` block to add the label to the axis once for each status. Right? Why not provide a legend?